tests/Unleash.Tests/Streaming/StreamingFeatureFetcherTests.cs

```diff
@@ -258,9 +258,13 @@ public async Task Receives_Hydration_Events_From_Sse_Server()
         timer.Stop();
 
         var enabled = unleash.IsEnabled("deltaFeature");
+        var backupPath = settings.GetFeatureToggleFilePath();
+        var fileContent = File.ReadAllText(backupPath);
 
         // Assert
         Assert.IsTrue(enabled, "Feature should be enabled after handling the message.");
+        Assert.NotNull(fileContent, "File content should not be null");
```

Comment:
maybe worth checking something meaningful like presence of a feature that we stored